Source/Scene/GltfLoader.js

```diff
@@ -682,20 +679,30 @@ function loadAttribute(
     return attribute;
   }
 
+  const loadAsTypedArray = loader._loadAttributesAsTypedArray;
+  const isInstancedTransformAttribute =
+    modelSemantic === InstanceAttributeSemantic.TRANSLATION ||
+    modelSemantic === InstanceAttributeSemantic.ROTATION ||
+    modelSemantic === InstanceAttributeSemantic.SCALE;
   const loadFor2D =
-    modelSemantic === VertexAttributeSemantic.POSITION &&
-    loader._loadPositionsFor2D &&
+    (modelSemantic === VertexAttributeSemantic.POSITION ||
+      isInstancedTransformAttribute) &&
+    loader._loadAttributesFor2D &&
     !frameState.scene3DOnly;
 
+  const loadBuffer = !loadAsTypedArray || loadFor2D;
+  const loadTypedArray =
```

Comment:
Since we're changing the buffers that are loaded here, the statistics will need to be updated here after #10397 is merged.

To do this, I suggest renaming `StatisticsPipelineStage -> PrimitiveStatisticsPipelineStage` and creating a similar `NodeStatisticsPipelineStage` for these changes. 

I think I also missed some details about counting instance attributes :grimacing: It should be very similar to `countGeometry()` in `StatisticsPipelineStage`.